src/shogun/mathematics/Seedable.h

```diff
@@ -62,7 +62,7 @@ namespace shogun
 			random::seed(object, m_seed);
 		}
 
-		int32_t m_seed;
+		int64_t m_seed;
```

Comment:
these lines should be changed as well to `int64_t`:
https://github.com/shogun-toolbox/shogun/blob/167880c1ea5bc62ec4906f48a86d72427eb9a014/src/shogun/mathematics/Seedable.h#L24
https://github.com/shogun-toolbox/shogun/blob/167880c1ea5bc62ec4906f48a86d72427eb9a014/src/shogun/mathematics/Seedable.h#L31
https://github.com/shogun-toolbox/shogun/blob/167880c1ea5bc62ec4906f48a86d72427eb9a014/src/shogun/mathematics/Seedable.h#L68